api/pkg/apps/seeder/static.go

```diff
@@ -517,12 +517,88 @@ var (
 		},
 	}
 
+	UGProtectionIncident = form.Form{
+		Controls: []form.Control{
+			{
+				Name: "locationOfIncident",
+				Type: form.Text,
+				Label: i18n.Strings{{"en", "Location of incident"}},
+			},
+			{
+				Name: "timeOfIncident",
+				Type: form.Time,
+				Label: i18n.Strings{{"en", "Time of incident"}},
+			},
+			{
+				Name: "reportedIncidentDate",
+				Type: form.Date,
+				Label: i18n.Strings{{"en", "Date incident reported"}},
+			},
+			{
+				Name: "receivedBy",
+				Type: form.Text,
+				Label: i18n.Strings{{"en", "Received by"}},
+			},
+			{
+				Name:    "vulnerability",
+				Type:    form.Dropdown,
+				Label:   i18n.Strings{{"en", "Vulnerability"}},
+				Options: []i18n.Strings{
+					{{"en", "Child at Risk"}},
+					{{"en", "Elder at Risk"}},
+					{{"en", "Single parent"}},
+					{{"en", "Separated Child"}},
+					{{"en", "Disability"}},
+					{{"en", "Woman at Risk"}},
+					{{"en", "Legal and physical protection"}},
+					{{"en", "Medical condition"}},
+					{{"en", "Pregnant/ lactating woman"}},
+				},
+				Validation: form.ControlValidation{
+					Required: true,
+				},
+			},
+			{
+				Name:  "incidentDescription",
+				Type:  form.Textarea,
+				Label: i18n.Strings{{"en", "Description of the Incident i.e Where, when, what, who involved"}},
+			},
+			{
+				Name:  "incidentHasBeenReportedToPolice",
+				Type:  form.Checkbox,
+				Label: i18n.Strings{{"en", "Has the incident been reported to police?"}},
+				CheckboxOptions: []form.CheckboxOption{
+					{
+						Label: i18n.Strings{{"en", "Has been reported?"}},
+						Value: "yes",
+					},
+				},
+			},
+			{
+				Name:  "comment",
+				Type:  form.Textarea,
+				Label: i18n.Strings{{"en", "Comment"}},
+			},
+			{
+				Name:    "incidentHasBeenReportedToOthers",
```

Comment:
I took the liberty here to leave out the "No" option and make the field not required.